source/globalCommands.py

```diff
@@ -2028,6 +2028,24 @@ def script_activateConfigProfilesDialog(self, gesture):
 	script_activateConfigProfilesDialog.__doc__ = _("Shows the NVDA Configuration Profiles dialog")
 	script_activateConfigProfilesDialog.category=SCRCAT_CONFIG
 
+	def script_toggleConfigProfileTriggers(self,gesture):
+		if config.conf.profileTriggersEnabled:
+			config.conf.disableProfileTriggers()
+			# Translators: The message announced when temporarily disabling all configuration profile triggers.
+			state = _("Configuration profile triggers disabled")
+		else:
+			config.conf.enableProfileTriggers()
+			# Explicitly trigger profiles for the current application.
+			mod=api.getForegroundObject().appModule
+			trigger = mod._configProfileTrigger = appModuleHandler.AppProfileTrigger(mod.appName)
+			trigger.enter()
+			# Translators: The message announced when re-enabling all configuration profile triggers.
+			state = _("Configuration profile triggers enabled")
+		ui.message(state)
+	# Translators: Input help mode message for toggle configuration profile triggers command.
+	script_toggleConfigProfileTriggers.__doc__=_("Disables or enables all configuration profile triggers. Disabling takes effect until NVDA is restarted")

```

Comment:
Maybe change this a bit:
"Toggles disabling of all configuration profile triggers. Disabling remains in effect until NVDA is restarted"